root/components/forms.tt

```diff
@@ -187,18 +187,48 @@
     [% END %]
 [%- END -%]
 
+[%- MACRO too_short_year_error(id) BLOCK -%]
+  <div class="error field-error" id="[% id %]" style="display: none">
+    [% WRAPPER form_row %]
+      [%- l('The year should have four digits (if before year 1000, pad with zeros, such as 0123)') -%]
```

Comment:
Maybe it’s just me but it sounds ambiguous whether “if” starts a new sentence or applies to the previous one. I would suggest to replace parenthesis with full sentence, and to be gentle. (Applies to both occurrences of this message.)
```suggestion
      [%- l('The year should have four digits. (If before year 1000, please pad with zeros, such as 0123.)') -%]
```

Reply:
Changed the comment a bit following your suggestions but with a wording that seems clearer to me, let me know what you think :)